src/js/index.js

```diff
@@ -38,3 +42,16 @@ iconCreate.addEventListener('click', () => {
   iconBookmark.classList.remove('active')
   iconCreate.classList.add('active')
 })
+
+// bookmark toggle
+
+bookmarks.forEach(bookmark => {
+  bookmark.addEventListener('click', () => {
+    bookmark.classList.toggle('fa')
+    console.log('Button was clicked')
```

Comment:
It is really good for working on code to use `console.log()`, however, it mostly is not needed after finishing the code and shouldn't be merged to main.